datasets/c2smsfloods.yaml

```diff
@@ -0,0 +1,17 @@
+Name: Cloud to Street - Microsoft Flood and Clouds Dataset
+Description: This dataset consists of chips of Sentinel-1 and Sentinel-2 satellite data. Each Sentinel-1 chip contains a corresponding label for water and each Sentinel-2 chip contains a corresponding label for water and clouds. Data is stored in folders by a unique event identifier as the folder name. Within each event folder there are subfolders for Sentinel-1 (s1) and Sentinel-2 (s2) data. Each chip is contained in its own sub-folder with the folder name being the source image id, followed by a unique chip identifier consisting of a hyphenated set of 5 numbers. All bands of the satellite data, as well as the labels, and overview images are contained within the chip folder.
+Documentation: https://www.drivendata.org/competitions/81/detect-flood-water/
+Contact: support@cloudtostreet.info
+ManagedBy: "[Radiant Earth Foundation](https://radiant.earth/)"
+UpdateFrequency: Not updated
+Tags:
+  - computer vision
```

Comment:
@kbgg are these stored as standard GeoTIFFs or COGs? If they are COGs and there is a STAC collection you can add the tags below:
- cog
- stac

Reply:
Good catch, added the cog tag. We don't have a publicly accessible static STAC catalog for this collection so the stac tag and explore example don't apply.